edx/edx-ora2
# [BD-05]: Add missing occurrences of pluggable editor

**TL;DR -** This implement a few missing pluggable editor methods in the `grades` and `leaderboard` sections.

JIRA: [BLENDED-783](https://openedx.atlassian.net/browse/BLENDED-783)

**What changed?**

- Pluggable submission editor is now loaded properly on the `Your grade` and `Top responses` sections

**Developer Checklist**


**Testing Instructions**

1. Checkout this branch.
2. Add a ora problem with self assessment only.
3. Enable the leaderboard (`top responses`).
4. Submit the problem and assess the response.
5. Check that the `your grade` section displays a correctly formatted reponse (no html code).
6. Check that the `top resposes` properly displays the response

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
